modules/server/templates/scripts/forseti-server/forseti_env.sh.tpl

```diff
@@ -1,5 +0,0 @@
-export SQL_PORT=${cloudsql_db_port}
-export SQL_INSTANCE_CONN_STRING=${project_id}:${cloudsql_region}:${cloudsql_instance_name}
```

Comment:
The PR for the initialize forseti services script currently depends on this script: https://github.com/forseti-security/terraform-google-forseti/pull/462/files#diff-7e160fa2f4b590ae541ebc8e4065dd4fR22.

Reply:
A new Terraform variable can be created and use in place of `SQL_INSTANCE_CONN_STRING`?

Reply:
Or can we directly use it's value:`SQL_PROXY_COMMAND+=" -instances=${project_id}:${cloudsql_region}:${cloudsql_instance_name}=tcp:${cloudsql_db_port}"`?

Reply:
Sure.  The cloudsql module is already being passed to the server.
https://github.com/forseti-security/terraform-google-forseti/blob/48343b66cd04778ef0c94a8a04ff7e0d51483848/main.tf#L167

and the cloudsql module outputs the connection string.

https://github.com/forseti-security/terraform-google-forseti/blob/48343b66cd04778ef0c94a8a04ff7e0d51483848/modules/cloudsql/outputs.tf#L16-L21